### DISCUSSION TITLE
Autoimport not working with neovim?

### DISCUSSION THREAD

**Discussion by igorlfs** [python-lsp/python-lsp-server#298] [Configuration]
Here's my config:
```lua
lspconfig.pylsp.setup({
    settings = {
        pylsp = {
            plugins = {
                rope_autoimport = {
                    enabled = true,
                },
            },
        },
    },
})
```
OS: Arch Linux
nvim version: NVIM v0.9.0 HEAD
pylsp version: 1.6.0
rope version: 1.4.0
pylsp-rope version: 0.1.10 (installed via AUR)

**Comment by ccordoba12** [comment]
@igorlfs, autoimport is not part of PyLSP 1.6.0. It will be of 1.7.0, to be released before the end of the year.

**Reply by igorlfs** [reply]
Oops, sorry! It is in the README, so I thought it was already merged (I mean shipped with 1.6.0)
